src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -329,6 +340,9 @@ instance (TypeHasDoc pm, IsoValue pm) => TypeHasDoc (ProposeParams pm) where
   typeDocHaskellRep = concreteTypeDocHaskellRep @(ProposeParams ())
   typeDocMichelsonRep = concreteTypeDocMichelsonRep @(ProposeParams ())
 
+instance HasAnnotation pm => HasAnnotation (ProposeParams pm) where
+  annOptions = defaultAnnOptions { fieldAnnModifier = dropPrefixThen toSnake }
```

Comment:
I think it makes sense to extract `defaultAnnOptions { fieldAnnModifier = dropPrefixThen toSnake }` into a top-level constant in this module since it's used in many places.